domdiv/main.py

```diff
@@ -971,17 +972,36 @@ def filter_sort_cards(cards, options):
     if options.expansion_dividers:
 
         cardnamesByExpansion = {}
+        randomizersByExpansion = {}
```

Comment:
There's a class called `Counter` in the standard library `collections` module that would clean up this code a bit (get rid of the `setdefaults` below etc.) Also, this should maybe be called `randomizerCountsByExpansion`.

Similarly, there's `defaultdict` in the same module that would let you define `cardnamesByExpansion` as `defaultdict(dict)` and avoid having to explicitly pre-fill new entries below.

Reply:
Nice!  I'll make the change to use these.